tests/foreman/ui/test_errata.py

```diff
@@ -221,46 +281,96 @@ def test_end_to_end(
         ],
         'module_stream_packages': [],
     }
-    assert _install_client_package(vm, FAKE_1_CUSTOM_PACKAGE)
+    _UTC_format = '%Y-%m-%d %H:%M:%S UTC'
+    # Capture product and repository with the desired content
+    assert len(product_list := module_target_sat.api.Product(organization=module_org).search()) > 0
+    _product = product_list[-1].read()
+    assert len(_product.repository) > 0
+    _repository = _product.repository[-1].read()
+    # Remove custom package if present, then install outdated version
+    _remove_client_package(registered_contenthost, FAKE_1_CUSTOM_PACKAGE_NAME)
+    assert _install_client_package(registered_contenthost, FAKE_1_CUSTOM_PACKAGE)
+    assert (
+        appl_errata := registered_contenthost.applicable_errata_count
```

Comment:
PRT just blows my mind here with `E       assert 1 == 1`
I though I understood how the walrus operator works, also locally this works. No idea what's wrong in PRT's python version walrus implementation. Maybe try without walrus.

Reply:
@JacobCallahan and I had some discussion around this, there must be some sort of ambiguity in the assignment I am not seeing. I see success assigning the `applicable_errata` first, and then arresting on the next line